### ISSUE TITLE
Does not build with latest servant

### ISSUE THREAD

**Issue by Fuuzetsu** [brendanhay/gogol#147]
gogol-core doesn't specify upper bound on servant version. As it happens, servant 0.17 hides the module `Servant.Utils.Links` that gogol-core tries to re-export, so the build fails.

**Comment by AlistairB** [comment]
Hi :wave: 

I believe this was fixed as part of #144 . However, there hasn't been a release to hackage since then. (Unfortunately there doesn't seem to be too much active maintenance on this project right now)

As such, I have been pulling it in via `extra-deps` with stack.

```yaml
  - git: https://github.com/brendanhay/gogol.git
    commit: 801f00b6e63c22522eb2b3e3e2fc77694730a70f
    subdirs:
      - core
      - gogol
      - gogol-logging
      - gogol-pubsub
      - gogol-cloudtrace
```
I think you can achieve similar things with `cabal.project` if you are just using cabal.

**Comment by Fuuzetsu** [comment]
OH, I didn't see #144 and I swear I looked. I don't know how I missed it, closing as duplicate.

Thanks!
